x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments/assignments.cy.ts

```diff
@@ -77,42 +67,23 @@ describe('Alert user assignment - ESS & Serverless', { tags: ['@ess', '@serverle
     });
 
     it('alert with some assignees in alerts table', () => {
-      const users = [ROLES.detections_admin, ROLES.t1_analyst];
```

Comment:
Is there any relationship between the cypress `defaultUser` and these roles? It seems like we're not only narrowing the scope of what these are testing (e.g. only role A where before we tested roles A and B). If we were, I would expect one of these two roles to be used here.

In my eyes this test previously was validating that "alert assignment works with the builtin roles that we document/prescribe for the feature." I'm not sure if that's still true, and to what extent.

Reply:
Not really. We were always running test cases in this file with `defaultUser` which is superuser. We were just assigning different users (with various roles) to alerts here, but always by superuser. The `loadPageAs(ALERTS_URL);` (or `login();` with the new changes) call in `beforeEach` signs in superuser account.

To test different roles and different subscription models we run test cases in other files in the folder here https://github.com/elastic/kibana/tree/main/x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments

<img width="617" alt="Screenshot 2024-02-23 at 14 58 51" src="https://github.com/elastic/kibana/assets/2700761/1103e6eb-eab5-4078-ac7d-e7cc9d63944c">

Reply:
Ahh, that's good to know! So there wasn't any RBAC being tested here, hence we don't really lose any functional testing. Thank you!